bricksrc/command.py

```diff
@@ -5,6 +5,7 @@
     "Command": {
         "tags": [TAG.Point, TAG.Command],
         "subclasses": {
+            "Boiler_Command": {"tags": [TAG.Boiler, TAG.Command, TAG.Point]},
```

Comment:
The `Boiler_Command` doesn't have a very descriptive definition --- what is the intended use? Like an `On_Off_Command` or `Enable_Command` on a Boiler?

Reply:
Boiler_Command is meant to be generic for any commands for a boiler like Fan_Command (compared with Fan_Speed_Command.) We might expand it to Boiler_On_Off_Command, Boiler_Power_Command, etc.